gcloud-java-datastore/src/main/java/com/google/gcloud/datastore/QueryResultsImpl.java

```diff
@@ -86,8 +89,8 @@ protected T computeNext() {
     if (!entityResultPbIter.hasNext()) {
       return endOfData();
     }
-    DatastoreV1.EntityResult entityResultPb = entityResultPbIter.next();
-    //cursor = entityResultPb.getCursor(); // only available in v1beta3
+    com.google.datastore.v1beta3.EntityResult entityResultPb = entityResultPbIter.next();
+    cursor = entityResultPb.getCursor();
```

Comment:
Is cursor will be available after the last next (and last batch). I assume so, @pcostell  to verify.

Also, I assume that a batches end_cursor is always going to be available (even for last batch) and that it would
be equal to the last entity result's cursor?